docker/cmsmon-rucio-ds/cronjobs.txt

```diff
@@ -0,0 +1,3 @@
+#
+# postStart K8s handler fills the /etc/environment file
+45 6 * * * . /etc/environment; /data/CMSSpark/bin/cron4rucio_datasets_daily_stats.sh --keytab /etc/secrets/keytab --cmsr /etc/secrets/cmsr_cstring --rucio /etc/secrets/rucio --amq /etc/secrets/amq-creds.json --cmsmonitoring /data/CMSMonitoring.zip --stomp /data/stomp-v700.zip --eos /eos/user/c/cmsmonit/www/rucio_daily_ds_stats >> /proc/$(cat /var/run/crond.pid)/fd/1 2>&1
```

Comment:
this are all hard-coded parameter which will not allow you to easily debug the issue. For instance, if you'll deploy the image it will only trigger crontab at 6:45, so you'll be stuck to wait till that time to debug it. If you'll keep it in k8s manifest file though you don't have this issue, since someone can easily modify k8s file to test if workflow is running at a time convenient during a test.